### DISCUSSION TITLE
Column with hyphens and spaces in forumla [Apr 09]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#1719] [Discord Threads]
> 🙋‍♂️ **SilentDigit**

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### Currently, we are improving this. Please stay tuned for upcoming releases.  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/961865133603303424)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
